python/ray/serve/_private/router.py

```diff
@@ -42,6 +42,7 @@
 from ray.util import metrics
 
 logger = logging.getLogger(SERVE_LOGGER_NAME)
+PUSH_METRICS_TO_CONTROLLER_TASK_NAME = "push_metrics_to_controller"
```

Comment:
ditto

Reply:
There currently isn't a metrics manager in router. However I think it is a good idea to add one - I will address this in a separate PR.